FileOnQ.Imaging.Heif.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.105
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.32014.148
```

Comment:
This project doesn't official support Visual Studio 2022, this statement should not be updated. The C++ code fails when using the compilers in Visual Studio 2022 and we only support 2019.